concepts/strings/introduction.md

```diff
@@ -5,17 +5,43 @@ A `string` in Go is an immutable sequence of bytes, which don't necessarily have
 A `string` literal is a sequence of bytes enclosed in double quotes:
 
 ```go
-name := "Jane"
+firstName := "Jane"
 ```
 
 Strings can be concatenated via the `+` operator:
 
 ```go
-fullName := "Jane" + " " + "Austen"
+fullName := "Jane" + " " + "Austen" // fullName = "Jane Austen"
 ```
 
-Some special characters need to be escaped with a leading backslash:
+Some special characters need to be escaped with a leading backslash, such as `\b` for Backspace and `\n` for new lines in strings.
```

Comment:
```suggestion
Some special characters need to be escaped with a leading backslash, such as `\t` for a tab and `\n` for a new line in strings.

```

Tabs are a more common special character than backspaces, so it makes for a better example.